Installation/CHANGES.md

```diff
@@ -45,6 +45,16 @@ Release date: September 2019
    the approximate dihedral angle between 2 vectors. Corresponding functors
    in the model (`Compute_approximate_angle_3`) and free function (`approximate_angle`)
    are also added.
+ - The following objects are now hashable and thus trivially usable
+   with `std::unordered_set` and `std::unordered_map`:
+   `CGAL::Aff_transformation_2.h`, `CGAL::Aff_transformation_3.h`,
+   `CGAL::Bbox_2.h`, `CGAL::Bbox_3.h`, `CGAL::Circle_2.h`,
+   `CGAL::Direction_2.h`, `CGAL::Direction_3.h`,
+   `CGAL::Iso_cuboid_3.h`, `CGAL::Iso_rectangle_2.h`,
+   `CGAL::Point_2.h`, `CGAL::Point_3.h`, `CGAL::Ray_2.h`,
+   `CGAL::Ray_3.h`, `CGAL::Segment_2.h`, `CGAL::Segment_3.h`,
+   `CGAL::Vector_2.h`, `CGAL::Vector_3.h`, `CGAL::Weighted_point_2.h`
+   and `CGAL::Weighted_point_3.h`.
```

Comment:
I think you did not want those `.h` extensions after the object types.

Reply:
Oops, yes, that was a badly processed copy-paste... I'll correct it.